src/lib/package-lib.js

```diff
@@ -25,6 +25,7 @@ module.exports = (pathToProject, entry, baseEslintPath, options = {}) => {
       filename: outputFilename,
       libraryTarget: options.libraryTarget ? 'umd' : undefined,
       globalObject: options.libraryTarget,
+      hashFunction: 'sha512' // remove when we upgrade to webpack 5
```

Comment:
I'm a little uncomfortable with this but I think it's fine, and appropriate until we get the webpack upgrade done.

Reply:
I think it might be worth updating this comment to say "change to xxhash64 when we upgrade to webpack 5".

I ended up down the rabbit-hole on this one!  Figured I would leave my findings here for anyone else reading this:

- webpack `4.x` [broke in Node `17.x`](https://github.com/webpack/webpack/issues/14532) because Node switched to OpenSSL3 and OpenSSL3 removed support for the md4 hash (which is what webpack `4.x` used by default).
- This is 'fixed' in [webpack 5.61.0](https://github.com/webpack/webpack/releases/tag/v5.61.0) when they added a special wasm md4 implementation. So, we could remove this config after upgrading to `v5.61.0+`.
- The [documentation for `hashFunction`](https://webpack.js.org/configuration/output/#outputhashfunction) suggests using `xxhash64` when on version `v5.54.0+` since that will be the default for webpack 6 (and it is faster)

Reply:
Great investigation, thanks! I've got an issue for upgrading to webpack 5, so I've linked to this comment from there to close the loop.